tests/functional/io_apis_test.c

```diff
@@ -0,0 +1,831 @@
+// Copyright 2018-2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * ----------------------------------------------------------------------------
+ * splinter_io_apis_test.c --
+ *
+ * This test exercises core IO interfaces that are used by SplinterDB
+ * to verify that these interfaces work as expected when executed:
+ *
+ * - Using threads, with Splinter using default malloc()-memory allocation
+ *
+ * The test design is quite simple:
+ *
+ * - Starting with a default IO configuration, create a device of some size
+ *   and using Sync IO interfaces driven by the main program, fill the
+ *   device out with some known data. Read back each page worth of data from
+ *   the file using sync-read and verify correctness. This case covers the
+ *   correctness check for basic sync read/write APIs driven by the main thread.
+ *
+ * - Using the data generated above, fire-up n-threads. Assign contiguous
+ *   sections of the file to each thread. Each thread verifies previously
+ *   written data using sync-read. Then, each thread writes new data to its
+ *   section using sync-writes, and verifies using sync-reads.
+ * ----------------------------------------------------------------------------
+ */
+
+#include "platform.h"
+#include "config.h"
+#include "io.h"
+#include "trunk.h" // Needed for trunk_get_scratch_size()
+#include "task.h"
+
+/*
+ * Structure to package arguments needed by test-case functions. This packaging
+ * allows to pass this set of args around. These are then supplied by worker
+ * functions invoked by pthreads to invoke the work-horse function(s).
+ */
+typedef struct io_test_fn_args {
+   platform_heap_id    arg_hid;
```

Comment:
We don't prefix fields in the rest of the code.  Please follow the general naming conventions.

Reply:
Renamed member fields as suggested.